### ISSUE TITLE
Add some way to update/overwrite file contents (e.g `updateFile(...)`)

### ISSUE THREAD

**Issue by lakscastro** [lakscastro/shared-storage#79]
## Feature request

Currently we can't overwrite/update the content of a given file we already have permissions over.

So the request is only to add an API which purpose is to receive a given `uri` and `content` or `bytes` as parameters and overwrite the content.

Sample usage:

```dart
final updatedFile = await updateFile(
  targetFile,
  bytes: UintList.fromList('content in bytes'.codeUnits),
  content: '...',
);
```

This will permit edit **only the file contents and not the file mime type.**

## Useful resources

- [how-to-properly-overwrite-content-of-file-using-android-storage-access-framework](https://stackoverflow.com/questions/56902845/how-to-properly-overwrite-content-of-file-using-android-storage-access-framework).

_Originally requested by @jfaltis at #61._

**Comment by lakscastro** [comment]
Would be helpful to use you implementation here, seems like it's exactly what we need here.

I'll open a PR to add your implementation here, or you can do by yourself if you want, just let me know.

Although I don't see reason to add `flutter_content_resolver` as dependency:

- It's easier to simply use your implementation.
- This other package is focused on content resolvers which are not our focus here: we just want to overwrite some files, content resolvers are just the tool.
- This will add more constraints to the feature here since the native code is defined by another package.
- Avoid possible future version and sdk conflicts.

Also let me know what do you think.

**Comment by jfaltis** [comment]
I agree with your points on not using the flutter_content_resolver as a dependency. I will open a pull request with the implementation in the upcoming days.

**Comment by lakscastro** [comment]
Closing as already implemented by #85.
